docs/search/getting-started.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 20
+title: Getting started with Search
+---
+
+In this getting started guide, you will add search functionality to the Address Book app that you build in the [Macrometa Quickstart](https://macrometa.com/docs/quickstart).
+
+## Prerequisites
+
+- A [Macrometa account](https://auth.paas.macrometa.io/).
+- Completion of our [quickstart guide](https://macrometa.com/docs/quickstart).
+
+## Step 1. Create a View
+
+ A _search view_ is an inverted index that contains the search configuration and all document attributes across one or many collections.
+
+1. On the side menu, click **Search Views**.
+2. Click **New View**.
```

Comment:
```suggestion
1. Click **New View**.
```

Reply:
We can change all ordered list items to `1.` so we can easily add/remove steps without changing the whole order manually